src/send-data.ts

```diff
@@ -23,14 +23,20 @@ const sendDataComment = async (url: string, prData: PRData): Promise<string> =>
   }
 
   try {
-    const response = await axios.post(url, postData, {
-      headers: { responseType: 'comment' },
-    });
+    const response = await axios.post(url, postData);
```

Comment:
```suggestion
    const { data } = await axios.post(url, postData);
```
might simplify things a *v tiny* bit if we de-structure this earlier...Maybe we could do `const { data: responseData } = await axios.post(url, postData);` so that the other lines become `responseData.data...` - feels weird having all of these `data.data` bits around.